helm/scripts/redis-presync.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -x  # Print each command
+set +e  # Disable immediate exit on error
+echo "Starting sso-presync hook..."
+# Source common functions
+source "$(dirname "$0")/postsync-lib.sh"
+
+
+# Create configmap
+create_or_update_configmap redis-config \
+  --from-literal=REDIS_PORT="$redis_port" \
```

Shellcheck (0.11.0)

[warning] redis_port is referenced but not assigned.

(SC2154)